src/script/InitDelegateeDeposits.s.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Multicall} from "@openzeppelin/contracts/utils/Multicall.sol";
+import {IGovLst, Staker} from "../interfaces/IGovLst.sol";
+
+/// @title InitDelegateeDeposits
+/// @author [ScopeLift](https://scopelift.co)
+/// @notice A script to initialize deposits for delegatee addresses in the GovLst contract.
+/// @dev This script reads delegatee addresses from a JSON file, checks if they already have deposits,
+/// and initializes deposits for those that don't have one yet using multicall batching for efficiency.
+abstract contract InitDelegateeDeposits is Script {
+  /// @notice Reference to the GovLst contract.
+  IGovLst govLst = IGovLst(getGovLstAddress());
+
+  /// @notice The default deposit ID value used to identify uninitialized deposits.
+  uint256 public immutable DEFAULT_DEPOSIT_ID = Staker.DepositIdentifier.unwrap(govLst.DEFAULT_DEPOSIT_ID());
+
+  /// @notice The number of operations to include in each multicall batch.
+  uint256 public immutable BATCH_SIZE = multicallBatchSize();
+
+  /// @notice Main entry point for the script.
+  /// @dev Reads addresses from file, filters those needing initialization, and processes them in batches.
+  function run() public virtual {
+    vm.startBroadcast();
+
+    address[] memory _delegateeAddresses = readDelegateeAddressesFromFile();
+    uint256[] memory _depositIds = getDepositIdsForDelegateeAddresses(_delegateeAddresses);
+    (address[] memory _addressesToInit, uint256 _numOfAddressesToInit) =
+      filterDelegateeAddresses(_delegateeAddresses, _depositIds);
+    (, uint256 _batchCount) = callFetchOrInitializeDepositForDelegatee(_addressesToInit, _numOfAddressesToInit);
+
+    console2.log("\n========== INITIALIZATION SUMMARY ==========");
```

Comment:
Console output looks good, but it shows up in our tests. We should have some kind of switch on the script contract like `bool showsSummaryOutput` that defaults to true, but can be overridden to false. In the tests, we should set it to false.

Reply:
added in https://github.com/withtally/stGOV/pull/91/commits/0d0640d8d1eadf39eb49192e2009ed33da2a2693